x/auth/client/testutil/suite.go

```diff
@@ -461,6 +462,8 @@ func (s *IntegrationTestSuite) TestCLIQueryTxsCmdByEvents() {
 				fmt.Sprintf("--%s=json", tmcli.OutputFlag),
 			},
 			false,
+			"transaction spanned more than two shares, this is not yet supported",
+			// TODO: change this to not expect an error when functionality is added to celestia-core
```

Comment:
Should we track this in an issue instead?

Reply:
Or, actually this is sufficient I think: https://github.com/celestiaorg/celestia-core/issues/718#issuecomment-1098158995